commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/SetAttribute.java

```diff
@@ -62,9 +102,28 @@ public static SetAttribute of(final Integer variantId, final String name, @Nulla
      * @return update action
      */
     public static SetAttribute ofUnsetAttribute(final Integer variantId, final String name) {
-        return of(variantId, name, null);
+        return ofVariantId(variantId, name, null);
     }
 
+    /**
+     * Action to remove a custom attribute.
+     * @param variantId the variant the attribute value should be unset
+     * @param name the name of the attribute, consult the product type to find the name
+     * @return update action
+     */
+    public static SetAttribute ofVariantIdUnsetAttribute(final Integer variantId, final String name) {
```

Comment:
Suggestion for another names: `ofUnsetAttributeForVariantId` and `ofUnsetAttributeForSku`?